packages/react/src/version.ts

```diff
@@ -0,0 +1,4 @@
+// @uifabric/react@0.1.0
```

Comment:
Guessing this file will be overwritten/corrected when the real version is generated, but wouldn't hurt to update it for initial checkin as well.

Reply:
maybe i should also change the scope name. :)